### ISSUE TITLE
Syntax Error Help

### ISSUE THREAD

**Issue by Mortem20** [Quatrecentquatre-404/cyberdrop-downloader#1]
Hello, i am not the biggest brain with python so please sorry for that. I tried to run py and cmd with the instruction you wrote but I get this errors. Do I something wrong? And sry for my english and the german cmd because I am german. 
![Screenshot (748)](https://user-images.githubusercontent.com/77106157/147854056-b9932c86-4a4f-45bf-889d-509ae58d9c05.png)
![Screenshot (749)](https://user-images.githubusercontent.com/77106157/147854057-e5714041-a6b5-4c9e-a8f7-1f59684281de.png)

**Comment by Quatrecentquatre-404** [comment]
Yeah, it's because you are pasting ``~$ `` which is not a part of the command. You have two solutions :

If you have git for Windows : You can download the git repository using this
If you don't have git or any emulator : Click on Download ZIP, the green button. Then, extract the folder on the desktop. Finally, complet steps from the README.md file.

**Comment by Mortem20** [comment]
So, I put the zip file on the desktop and tried both once to execute the line in Git, as well as in CMD. In CMD you can see that I get a fatal message, in Git the line worked, but when I open the Test Python file, there is nothing in it even if I scroll all the way down. If I open it with Git I get an access denied message...

![Screenshot (750)](https://user-images.githubusercontent.com/77106157/147885360-7c0e2105-41fa-4dd6-ae2f-00eebd96d80e.png)
![Screenshot (751)](https://user-images.githubusercontent.com/77106157/147885362-224245d7-d053-4924-ae20-ca7fedb78cc9.png)

**Comment by Quatrecentquatre-404** [comment]
Go to your cyberdrop-downloader directory and create a ``downloads`` file. Then, launch ``tests.py``

**Comment by Mortem20** [comment]
What kind of file? A txt file? Or a new folder? Because the downloads folder with the meta file is already installed. 
Sorry for my stupidness

**Comment by Quatrecentquatre-404** [comment]
I meant create a downloads folder. But it's already supposed to be here. If there is the meta file, there is the files folder. Go ahead and find the downloaded resources.
